server/src/metakb/query.py

```diff
@@ -12,7 +12,15 @@
     VariantPrognosticStudyStatement,
     VariantTherapeuticResponseStudyStatement,
 )
-from ga4gh.va_spec.base import Direction, Document, EvidenceLine, Method, TherapyGroup
+from ga4gh.va_spec.base import (
+    Direction,
+    Document,
+    EvidenceLine,
+    MembershipOperator,
```

Comment:
this is a really interest change. is the concept of "membership" used anywhere else besides Combination Therapies?

Reply:
It's used for both TherapyGroups: Combination (MembershipOperator.AND) and Substitution (MembershipOperator.OR)

Reply:
or rather I guess my question is whats the motivation in va spec for generalizing membership

Reply:
We also use it in Condition Sets

Reply:
https://github.com/ga4gh/va-spec/issues/252#issuecomment-2631684482